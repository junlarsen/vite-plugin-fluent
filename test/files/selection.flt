cookies = I have baked { $count ->
  [one] one cookie
  *[other] { $count } cookies
} for you