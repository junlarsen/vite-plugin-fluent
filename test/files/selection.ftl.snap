import type { FluentResource, FluentBundle, FluentVariable } from "@fluent/bundle";
export declare const resource: FluentResource;
export declare type MessageSet = {
    readonly "cookies": {
        readonly "count": "one" | "other";
    };
};
export declare type MessageId = keyof MessageSet;
export declare function formatMessage<K extends MessageId>(bundle: FluentBundle, id: K, args: MessageSet[K], error?: null | Error[]): string;
